src/pages/index.js

```diff
@@ -20,80 +20,222 @@ import {
   previewImageModal,
   previewImage,
   previewImageTitle,
+  profileAvatar,
+  profileName,
+  profileSubtitle,
+  deleteCardModal,
+  deleteButton,
+  avatarEditButton,
+  avatarEditForm,
+  initialCards,
+  avatarProfile,
+  cardLikeButton,
 } from "../utils/constants.js";
+import api from "../components/Api.js";
 
-//userinfo
-
+// User info setup
 const userInfo = new UserInfo({
   nameSelector: ".profile__title",
   jobSelector: ".profile__subtitle",
 });
 
-const profilePopup = new PopupWithForm("#profile-edit-modal", (formData) => {
-  userInfo.setUserInfo({
-    name: formData["profile-title-input"],
-    job: formData["profile-subtitle-input"],
-  });
-});
+const cardSection = new Section(
+  {
+    items: [],
+    renderer: (cardData) => {
+      const cardElement = createCard(cardData);
+      cardSection.addItem(cardElement);
+    },
+  },
+  ".cards__list"
+);
+// cardSection.renderItems();
+let currentUserId;
+// Fetch user info and cards from the API
+api
+  .getAppData()
+  .then(([userData, cards]) => {
+    currentUserId = userData._id;
 
-const cardPopup = new PopupWithForm("#card-add-modal", (formData) => {
-  renderCard({
-    name: formData["card-title-input"],
-    link: formData["card-link-input"],
+    // Update user info in the DOM
+    userInfo.setUserInfo({
+      name: userData.name,
+      job: userData.about,
+    });
+
+    profileAvatar.src = userData.avatar;
```

Comment:
`    profileAvatar.src = userData.avatar;
`
Only methods of `UserInfo` should set/get the profile data (including the **avatar**)